cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -418,17 +426,23 @@ def __init__(self, *component_gates: cirq.Gate) -> None:
             else:
                 self.component_gates += (gate,)
 
-    def qubit_index_to_gate_and_index(  # pylint: disable=missing-function-docstring
-        self, index: int
-    ) -> Tuple[cirq.Gate, int]:
+    def _qubit_index_to_gate_and_index(self, index: int) -> Tuple[cirq.Gate, int]:
         for gate in self.component_gates:
             if gate.num_qubits() > index >= 0:
                 return gate, index
             index -= gate.num_qubits()
         raise ValueError("index out of range")
 
     def qubit_index_to_equivalence_group_key(self, index: int) -> int:
-        indexed_gate, index_in_gate = self.qubit_index_to_gate_and_index(index)
+        """Returns a key that differs between qubits in different component gates and
```

Comment:
Per dev guide, the docstrings should be one sentence.
https://google.github.io/styleguide/pyguide.html#:~:text=A%20docstring%20should,for%20docstrings%20below.

Reply:
Isn't it one sentence already?

Reply:
Sorry, not only one sentence, but one _line_.

> A docstring should be organized as a summary line (one physical line not exceeding 80 characters) terminated by a period, question mark, or exclamation point. When writing more (encouraged), this must be followed by a blank line, followed by the rest of the docstring starting at the same cursor position as the first quote of the first line.

Reply:
Ah yes, good. Done.